scylla/src/transport/topology.rs

```diff
@@ -184,6 +184,10 @@ enum PreCqlType {
         type_: PreCollectionType,
     },
     Tuple(Vec<PreCqlType>),
+    Vector {
+        type_: Box<PreCqlType>,
+        size: u16,
+    },
```

Comment:
Why did you choose specifically `u16` for size? Can you share your reference documentation?

Reply:
Good point, Ive now added the reference as a comment above that line.